src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -38,23 +38,27 @@ const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = ({
   )
 
   useEffect(() => {
-    // Update dynamic parameters when responses change
-    const JWT = localStorage.getItem(globalConfig.localStorageJWTKey)
-    Object.entries(parameterExpressions).forEach(([field, expression]) => {
-      evaluateExpression(expression as EvaluatorNode, {
-        objects: {
-          responses: { ...allResponses, thisResponse: response?.text },
-          currentUser,
-          applicationData,
-          functions,
-        },
-        APIfetch: fetch,
-        graphQLConnection: { fetch: fetch.bind(window), endpoint: graphQLEndpoint },
-        headers: { Authorization: 'Bearer ' + JWT },
-      }).then((result: any) =>
-        setEvaluatedParameters((prevState) => ({ ...prevState, [field]: result }))
-      )
-    })
+    // Update dynamic parameters when responses change, but only if there's no
+    // saved evaluations (usually INFORMATION type), or "showLiveParameters" has
+    // been explicitly set
```

Comment:
Prevents application evaluation queries from re-running during review/summary. Unfortunately, the "Information" type elements don't store a response, so we can used their previously evaluated values, so they still run in Summary view. (I might change this behaviour at some point)